rules/windows/defense_evasion_defender_exclusion_via_powershell.toml

```
@@ -70,6 +70,8 @@ Microsoft Windows Defender is an antivirus product built into Microsoft Windows.
 """
 references = [
     "https://www.bitdefender.com/files/News/CaseStudies/study/400/Bitdefender-PR-Whitepaper-MosaicLoader-creat5540-en-EN.pdf",
+    "https://www.elastic.co/security-labs/elastic-security-uncovers-blister-malware-campaign",
+    "https://www.elastic.co/security-labs/operation-bleeding-bear",
 ]
```

ghostengine also disables Windows defender: 

```suggestion
references = [
    "https://www.bitdefender.com/files/News/CaseStudies/study/400/Bitdefender-PR-Whitepaper-MosaicLoader-creat5540-en-EN.pdf",
    "https://www.elastic.co/security-labs/elastic-security-uncovers-blister-malware-campaign",
    "https://www.elastic.co/security-labs/operation-bleeding-bear",
    "https://www.elastic.co/security-labs/invisible-miners-unveiling-ghostengine"
]
```